client/lib/signup/flow-controller.js

```diff
@@ -159,7 +159,21 @@ assign( SignupFlowController.prototype, {
 	},
 
 	_processStep: function( step ) {
-		const dependencies = steps[ step.stepName ].dependencies || [];
+		let dependencies = steps[ step.stepName ].dependencies || [];
```

Comment:
Not minor at all. This was a leftover from a previous iteration of the code. Fixed.